I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 65, in script call
    call ch5
  File "game/ch5.rpy", line 138, in script
    show moricg1 with fade (1.0)
TypeError: 'MultipleTransition' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 65, in script call
    call ch5
  File "game/ch5.rpy", line 138, in script
    show moricg1 with fade (1.0)
  File "renpy/ast.py", line 1438, in execute
    renpy.exports.with_statement(trans, paired)
  File "renpy/exports.py", line 1682, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "renpy/display/core.py", line 2718, in do_with
    clear=clear)
  File "renpy/display/core.py", line 3276, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3633, in interact_core
    new_widget=layers_root)
TypeError: 'MultipleTransition' object is not callable

Windows-7-6.1.7601-SP1
Ren'Py 7.4.8.1895
Tri City Monsters 1.0
Thu Sep 30 09:44:27 2021
